addons/settings/fnc_init.sqf

```diff
@@ -166,7 +166,7 @@ GVAR(default) setVariable [_setting, [_defaultValue, _setting, _settingType, _se
 private _settingInfo = GVAR(userconfig) getVariable _setting;
 
 if (isNil "_settingInfo") then {
-    private _settingsHash = profileNamespace getVariable [QGVAR(hash), HASH_NULL];
+    private _settingsHash = GET_SERVER_NAMESPACE getVariable [QGVAR(hash), HASH_NULL];
```

Comment:
As this runs on client I think the macro could use some better name than `GET_SERVER_NAMESPACE` but not sure what name.